pennylane/operation.py

```diff
@@ -3039,6 +3040,20 @@ def convert_to_opmath(op):
     return op
 
 
+@contextmanager
+def use_new_opmath():
```

Comment:
i'm fine with leaving this but i'm not sure if anyone wants it anymore... @trbromley @Qottmann ?

Reply:
I don't think this context manager is used outside of this PR.